code/components/jomjol_flowcontroll/ClassFlowControll.cpp

```diff
@@ -195,7 +189,7 @@ void ClassFlowControll::SetInitialParameter(void)
     flowpostprocessing = NULL;
     disabled = false;
     aktRunNr = 0;
-    aktstatus = "Booting ...";
+    aktstatus = "Flow not started";
```

Comment:
is it `not started` or could we say `not yet started`?

Reply:
states renamed